docker/opg-s3-antivirus-update/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM amazonlinux:2 AS build-clamav
+
+RUN yum update -y &&\
+  amazon-linux-extras install epel -y &&\
+  yum install clamav clamd -y
+
+FROM golang:1.17.6 AS build-env
+
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/main ./cmd/opg-s3-antivirus-update
```

Comment:
Given that they're basically the same except this line, I suspect we could combine these two Dockerfiles, but probably worth saving that until we sort out the base image